### PR TITLE
Added brief description of commands on files page.

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

File: docs/services/files.rst [hunk 1]
```diff
@@ -1,105 +1,85 @@
...
 ^^^^^^^^
-Usage::
+Retrieves a list of of containers::
 
     rack files container list [optional flags]
 
 ``create``
 ^^^^^^^^^^
-Usage::
+Creates a container::
```
Comment (rgbkrk) [inline docs/services/files.rst]: *Create

File: docs/services/files.rst [hunk 2]
```diff
@@ -1,105 +1,85 @@
...
+Retrieves a list of containers. Optional flags can
+be used to refine your search::
 
     rack files container get --name <containerName> [optional flags]
     (echo containerName1 && echo containerName2) | rack files container get --stdin name [optional flags]
 
 ``update``
 ^^^^^^^^^^
-Usage::
+Creates or updates metadata for a specified container::
```
Comment (rgbkrk) [inline docs/services/files.rst]: Tense should be:

> Create or update metadata for a specified container

File: docs/services/files.rst [hunk 3]
```diff
@@ -1,105 +1,85 @@
...
+    rack files <resource> <action> [command flags]
 
-Commands
---------
 
 ``container``
 ~~~~~~~~~~~~~
 
-  Files Container operations
+All cloud files container operations use this syntax::
```
Comment (rgbkrk) [inline docs/services/files.rst]: This is becoming too verbose.

File: docs/services/files.rst (hunk 2 above)
Comment (DonNateR) [inline docs/services/files.rst]: I'll go ahead and change it

File: docs/services/files.rst [hunk 4]
```diff
@@ -108,50 +88,26 @@ Usage::
 ``upload-dir``
 ^^^^^^^^^^^^^^
 
-Usage::
+Uploads an object directory into a specified container::
```
Comment (rgbkrk) [inline docs/services/files.rst]: :+1:

File: docs/services/servers.rst [hunk 5]
```diff
@@ -4,12 +4,12 @@
 Servers
 =======
 
-This section gives a list of all Cloud Server commands supported by Rackspace CLI.
+This section lists all cloud servers commands supported by Rackspace CLI.
```
Comment (rgbkrk) [inline docs/services/servers.rst]: :+1:

File: docs/services/files.rst (hunk 3 above)
Comment (DonNateR) [inline docs/services/files.rst]: Operations actually should be "commands", I'll go ahead and make the change

File: docs/services/servers.rst [hunk 6]
```diff
@@ -4,12 +4,12 @@
...
 
 Commands
 --------
 
-All cloud server commands are based off of this syntax::
+All cloud server commands are based on this syntax::
```
Comment (rgbkrk) [inline docs/services/servers.rst]: Much improved.

**Pull request merged**